cabal.project

```diff
@@ -41,7 +41,7 @@ test-show-details: streaming
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
-  tag: b1eb0e678b834ef5a6eaea84e75b41a14123331a
+  tag: 631cb6cf1fa01ab346233b610a38b3b4cba6e6ab
```

Comment:
I am not sure if such change is really worth the effort, since `b1eb0e678b834ef5a6eaea84e75b41a14123331a` is part of `cardano-base` `master` branch as much as the merge commit is.

Reply:
The effort was minimal, and someone else said that others might be confused. I'm happy either way :shrug: